src/locale/pt-br.js

```diff
@@ -0,0 +1,12 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'pt-br',
+  weekdays: 'Domingo_Segunda-feira_Terça-feira_Quarta-feira_Quinta-feira_Sexta-feira_Sábado'.split('_'),
+  months: 'Janeiro_Fevereiro_Março_Abril_Maio_Junho_Julho_Agosto_Setembro_Outubro_Novembro_Dezembro'.split('_'),
+  ordinal: n => n
```

Comment:
should this be `ordinal : '%dº'` ?  Not sure.

Reply:
@iamkun The ordinal can be used or not. Here you can check examples that show that both ways are used in Brazil:
https://en.wikipedia.org/wiki/Date_and_time_notation_in_Brazil
http://www.maujor.com/w3c/datetime.html Here search for _Exemplos_ (Examples)

Reply:
After checking locale files in other JS date libraries, seems most are `nº`. But still it's up to you, native speaker. 😁

Reply:
@iamkun I've amended the commit with the fix. 
Thanks for the review!

Reply:
great THX